src/app/search/search.component.ts

```diff
@@ -267,15 +267,29 @@ export class SearchComponent implements OnInit {
 
   applyFilter(row: any) {
     this.model.query += this.parseFilter(row);
+    console.log(this.parseFilter(row))
```

Comment:
Are we keeping this?

Reply:
@nguyenLisa Yes. that's part of the function that formats the filter so it is acceptable by the server

Reply:
@iamebuka , I meant: is it useful to log this information to the console? If not, then there's no need for this.

Reply:
Oh yea it isn't necessary. I am removing it

Reply:
Great, thanks! With additional test cases, this should be good to go.

Reply:
@nguyenLisa I noticed some git error in my commit, I would like to make changes before you continue

Reply:
Please disregard this pull request, i realized i committed changes i was working on a different branch to this issue branch before pushing to git. Apart from that everything works fine. So i would be closing this and deleting the branch while i issue a new pull request just for this issue